src/app.js

```diff
@@ -708,7 +708,7 @@ var run = function() {
 			cb(files[fileNameFromKey(SOL_CACHE_FILE)]);
 			return;
 		}
-		var importRegex = /import\s*[\'\"]([^\'\"]+)[\'\"];/g;
+		var importRegex = /^\s*import\s*[\'\"]([^\'\"]+)[\'\"];/g;
```

Comment:
Haha! I had the exact same issue an hour ago 😃 

See #28. I like your change better as it is more flexible, will remove that part from mine.